src/ApiService/ApiService/Functions/Tool.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System.IO;
+using System.Net;
+using System.Reflection;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Azure.Functions.Worker.Http;
+
+namespace Microsoft.OneFuzz.Service.Functions;
+
+public class Tools {
+    private readonly IOnefuzzContext _context;
+    private readonly IEndpointAuthorization _auth;
+
+    public Tools(IEndpointAuthorization auth, IOnefuzzContext context) {
+        _context = context;
+        _auth = auth;
+    }
+
+    private static string ReadResource(Assembly asm, string resourceName) {
+        using var r = asm.GetManifestResourceStream(resourceName);
+        if (r is null) {
+            return "unknown";
+        }
+
+        using var sr = new StreamReader(r);
+        return sr.ReadToEnd().Trim();
```

Comment:
read resource here means read manifest ?

Reply:
does not seem to be called by anything

Reply:
forgot the remove after copying the code